app/upgrades/v16/upgrades.go

```diff
@@ -12,18 +12,34 @@ import (
 const (
 	// DAI/OSMO pool ID
 	// https://app.osmosis.zone/pool/674
+	// Note, new concentrated liquidity pool
+	// swap fee is initialized to be the same as the balancers pool swap fee of 0.2%.
 	daiOsmoPoolId = uint64(674)
-	// TODO: make sure this is what we desire.
+	// Denom0 translates to a base asset while denom1 to a quote asset
+	// We want quote asset to be DAI so that when the limit orders on ticks
+	// are implemented, we have tick spacing in terms of DAI as the quote.
 	desiredDenom0 = "uosmo"
 	// TODO: confirm pre-launch.
 	tickSpacing = 1
-
-	// TODO: confirm that concentrated pool swap fee should equal balancer swap fee.
 )
 
 var (
 	// TODO: confirm pre-launch.
 	exponentAtPriceOne = sdk.OneInt().Neg()
+
+	DAIIBCDenom  = "ibc/0CD3A0285E1341859B5E86B6AB7682F023D03E97607CCC1DC95706411D866DF7"
+	USDCIBCDenom = "ibc/D189335C6E4A68B513C10AB227BF1C1D38C746766278BA3EEB4FB14124F1D858"
+
+	// authorized_quote_denoms quote assets that can be used as token1
+	// when creating a pool. We limit the quote assets to a small set
+	// for the purposes of having convinient price increments stemming
+	// from tick to price conversion. These increments are in a human
+	// understandeable magnitude only for token1 as a quote.
+	authorizedQuoteDenoms []string = []string{
+		"uosmo",
+		DAIIBCDenom,
+		USDCIBCDenom,
```

Comment:
Is there no other desired quote denoms? WBTC and WETH feel like ones that would be rather "default". I know this is changeable, but would be nice to have the one's we intend to use already added

Reply:
These have been confirmed by Sunny as desired for launch. 

Let's not block PR on discussions and update this separately if any updates please